internal/decryption/keyproviders/key_provider.go

```diff
@@ -41,15 +40,15 @@ type KeyStoreIdentifier struct {
 	Provider KeyStoreProvider
 }
 
-func provideInput(prompt, defaultInput string) (string, error) {
-	if strings.TrimSpace(strings.ToLower(os.Getenv("CI"))) == "true" {
-		return defaultInput, nil
+func provideInput(prompt string) (string, error) {
```

Comment:
the linter somehow started to complain `defaultInput` was always `""` so removing as an arg